Cargo.toml

```diff
@@ -14,3 +14,10 @@ sourmash = "0.15.1"
 anyhow = "1.0.89"
 log = "0.4.22"
 env_logger = "0.11.5"
+
+# For JSON serialization/deserialization
+serde = { version = "1.0", features = ["derive"] }
```

Comment:
why version 1? latest is 1.0.210. (presumably dependabot will upgrade, just curious.)

Reply:
No good reason, just inherited from example code.